src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -269,6 +274,10 @@ export class SettingsEditor2 extends EditorPane {
 		if (ENABLE_LANGUAGE_FILTER && !SettingsEditor2.SUGGESTIONS.includes(`@${LANGUAGE_SETTING_TAG}`)) {
 			SettingsEditor2.SUGGESTIONS.push(`@${LANGUAGE_SETTING_TAG}`);
 		}
+
+		extensionManagementService.getInstalled(ExtensionType.System).then(extensions => {
```

Comment:
This is a similar hack that PreferencesSearchService uses to "unwrap" the result from the async API.